AuthorYYMMDD_fignr_001.yaml

```diff
@@ -0,0 +1,74 @@
+# This file can exist with the "source" information only, the entry data exists should be marked as "False" 
+# to indcate that neither the metadata nor the actual data were digitized or provided.
+# use "null" to idicate that a value is not available in the literature
+version: 1 # Is this necessary. Isn't this anyway given by git versioning
+
+source: 
+    doi: https://doi.org/10.1016/0039-6028(85)90985-9
+    bib id: Beden1985 # First Author and YYYY. Cross check for duplicats in the bib file.
+    figure: 2b # depending on the subfigure labels use 1, 1b, or 1_3
+    curve identifier: First cycle # use a unique short description to identify the curve in the figure
+    data exists: True

```

Comment:
What is the meaning of this exactly?

Reply:
I thought that a file can exist with `source` data only, in the case that you only want to create the yaml file but don't have the time to create the digitized plot (or the csv data). But I guess one can also simply query if the other entries in the file exist or not.

Reply:
```suggestion
```